distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -11,153 +11,118 @@
+. Change the property `SOLR_REQUIRE_TWO_WAY_TLS` to `SOLR_REQUIRE_TWO_WAY_TLS=false`
+. Restart ${branding}
+
+.Configuring Solr Transport Layer Security
+The default configuration does not use
+Transport Layer Security (TLS) between ${branding} and Solr. TLS adds overhead to Solr interactions
+and should not be necessary because Solr and ${branding} are running on the same host.
+To enable TLS:
 
 . Edit the `${home_directory}/etc/system.properties`
-.. Comment out the Solr Client Configuration for *Cloud Solr Client* and *Embedded Solr Client* sections.
-.. Uncomment the section for the *Http Solr Client*.
-
-[NOTE]
-====
-To point to an External Solr server instead of the default Internal one, put the url in `solr.http.url` (see below for example).
-Also, two jar files need to be added to the Solr installation as described in <<Installing Solr Cloud,Installing Solr Cloud>>.
-====
-
-.system.properties
-----
-
-solr.client = HttpSolrClient
-solr.http.url = https://solr-host-name:8993/solr
-solr.data.dir = ${karaf.home}/data/solr
-
-----
-
-.[[_http_solr]]Embedded Solr
-The Embedded Solr is an embedded, local Apache Solr server instance.
-It is a local instance that is a lightweight Catalog Provider solution.
-However, it does not provide a Solr Admin UI or a REST-like HTTP/XML and JSON API.
-If that is necessary, see <<HTTP Solr,HTTP Solr>> or <<Solr Cloud,Solr Cloud>>.
-
-.Embedded Solr Deployment
-[ditaa,embedded-solr-diagram,png]
-....
- Container (JVM)
- +=-----------------+
- |                  |
- | /--------------\ |
- | |cDEF          | |
- | | DDF          | |
- | |              | |
- | |  +---------+ | |
- | |  |Solr cFF0| | |
- | |  +---------+ | |
- | |              | |
- | \--------------/ |
- |                  |
- |                  |
- +------------------+
-....
-
-.When to Use the Embedded Solr
-The Embedded Solr configuration is great for demonstrations, training exercises, or for sparse querying and ingesting. See below for details.
-
-.Embedded Solr Tradeoffs
-* Requires no installation and very little configuration.
-* No HTTP connection required. As a result, there is no network overhead.
-* Not scalable.
-* There is no Solr Admin UI.
-* Backup must be performed manually.
-
-.Installing Embedded Solr
-
-No installation required. Follow the Configuring section below and restart the {branding}.
-
-.Configuring Embedded Solr
+. Change the property `solr.http.url` to read `https://` instead of `http://`
+. Restart ${branding}
+
+.Require Two-way (Client) Authentication
+This option is only meaningful if Solr and ${branding} communicate via the HTTPS protocol.
+The default configuration is secure and requires client authentication
+if the HTTPS protocol is used.
+
+Two-way authentication means the server (Solr) must trust the client (${branding}). If Solr
+did not trust ${branding}, Solr would reject all requests
+from ${branding} and all other the unauthenticated clients. Internally managed Solr and
```

Comment:
✏️ ❓ _Solr would reject all requests from ${branding} and all other unauthenticated clients_